app/gateway/changelog.md

```diff
@@ -9,6 +9,39 @@ Changelog for supported Kong Gateway versions.
 
 For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).
 
+## 3.4.2.0 
+**Release date** 2023/11/08
+
+### Features
+#### Enterprise
+
+* License management:
+  * Implemented a new grace period that lasts 30 days from the Kong Enterprise license expiration date. 
+    During the grace period all open source functionality will be available, and
+    Enterprise functionality will be set to read-only mode.
+  * Added support for counters such as routes, plugins, licenses, and deployment information to the license report.
+  * Added a checksum to the output of the license endpoint.
+### Fixes
+#### Core
+* DNS
+  * Fixed an issue with the DNS client was not adhering to configured timeouts in a predictable manner. [#11386](https://github.com/Kong/kong/pull/11386)
+  * Fixed an issue that caused the DNS client to resolve incorrectly during transient network and DNS server failures. [#11386](https://github.com/Kong/kong/pull/11386)
+* The default value of the [`dns_no_sync`](/gateway/3.5.x/reference/configuration/#dns_no_sync) option has been changed to `on`.
+[#11871](https://github.com/kong/kong/pull/11871).
+#### Plugins
+* [**OpenTelemetry**](/hub/kong-inc/opentelemetry/) (`opentelemetry`): Fixed an issue that resulted in traces with invalid parent IDs when balancer instrumentation was enabled. [#11830](https://github.com/Kong/kong/pull/11830)
+* [**OpenID Connect**](/hub/kong-inc/openid-connect/) (`openid-connect`): Fixed an issue with `using_pseudo_issuer`, where once propogated it was not used. [#6981](https://github.com/Kong/kong-ee/pull/6981)
```

Comment:
```suggestion
* [**OpenID Connect**](/hub/kong-inc/openid-connect/) (`openid-connect`)
  * Fixed an issue with `using_pseudo_issuer`, where it was not used after it was propagated.
```
We can't add links to kong-ee in the changelog.

Not sure about this rephrase, you're welcome to adjust.